app/views/recipes/results.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= render partial: "layouts/search_bar" %>
+
+<section class="section-body" id="results">
+<h2> Results for "<%= @query %>"</h2>
+
+
+<% if @recipes.empty? %>
+  <h3> No results found. </h3>
+<% else %>
+  <ul>
+  <% @recipes.each do |recipe| %>
+    <li>
+      <%= image_tag recipe.photo, alt: recipe.name %>
```

Comment:
I suggest making the `image_tag` also a link because it's human nature to want to click on the images.